Statistics/Distributions/LogNormal.cs

```diff
@@ -9,121 +9,111 @@ namespace Statistics.Distributions
         internal IRange<double> _ProbabilityRange;
-        private MathNet.Numerics.Distributions.LogNormal _Distribution;
-        private bool _Constructed;
 
-        #region Properties
-        public IDistributionEnum Type => IDistributionEnum.LogNormal;
+        #region IDistribution Properties
+        public IDistributionEnum Type => IDistributionEnum.Normal;
         [Stored(Name = "Mean", type = typeof(double))]
-        public double Mean { get; set; }
-
-        public double Median => _Distribution.Median;
-
-        public double Mode => _Distribution.Mode;
-
-        public double Variance => _Distribution.Variance;
-        [Stored(Name = "St_Dev", type = typeof(double))]
-        public double StandardDeviation{ get; set; }
-
-        public double Skewness => _Distribution.Skewness;
-
-        public IRange<double> Range { get; set; }
-        [Stored(Name = "Min", type = typeof(double))]
-        public double Min
-        {
-            get; set;
-        }
+         public double Mean { get{return _mean;} set{_mean = value;} }
+        [Stored(Name = "Standard_Deviation", type = typeof(double))]
+        public double StandardDeviation { get{return _standardDeviation;} set{_standardDeviation = value;} }
+        [Stored(Name = "Min", type =typeof(double))]
+        public double Min { get{return _min;} set{_min = value;} }
         [Stored(Name = "Max", type = typeof(double))]
-        public double Max
-        {
-            get; set;
-        }
+        public double Max { get{return _max;} set{_max = value;} }
         [Stored(Name = "SampleSize", type = typeof(Int32))]
         public int SampleSize { get; set; }
         [Stored(Name = "Truncated", type = typeof(bool))]
         public bool Truncated { get; set; }
+        #endregion
+        #region IMessagePublisher Properties
         public IMessageLevels State { get; private set; }
-        public IEnumerable<IMessage> Messages { get; private set; }
+        public IEnumerable<Utilities.IMessage> Messages { get; private set; }
+        #endregion
+
         #endregion
 
         #region Constructor
         public LogNormal()
         {
-            //for reflection
-            _Distribution = new MathNet.Numerics.Distributions.LogNormal(1, 1);
+            //for reflection;
+            Mean = 0;
+            StandardDeviation = 1.0;
+            _ProbabilityRange = IRangeFactory.Factory(0.0, 1.0);
+            Min = InverseCDF(0.0000000000001);
+            Max = InverseCDF(1-0.0000000000001);
+            State = Validate(new Validation.LogNormalValidator(), out IEnumerable<Utilities.IMessage> msgs);
+            Messages = msgs;
         }
-        public LogNormal(double mean, double standardDeviation, int sampleSize = int.MaxValue)
+        public LogNormal(double mean, double sd, int sampleSize = int.MaxValue)
         {
             Mean = mean;
-            StandardDeviation = standardDeviation;
+            StandardDeviation = sd;
```

Comment:
why do sd instead of standardDeviation? may we please code using words for readability?

Reply:
yes feel free to change back to standarddeviation